schema-updates.sql

```diff
@@ -0,0 +1,57 @@
+-- DO NOT IMPLEMENT YET .. trying to figure out why this happened. Running again.
```

Comment:
@sgoggins Is this supposed to be merged in?

Reply:
@ABrain7710 : These are changes I made to correspond with changes in the alembic script .. or to signal necessary changes to the alembic script. I am not sure how to write those yet.